index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./src/TimeSelect.jsx');
```

Comment:
I've noticed linting only requires `use strict` if your code has a function in it